ynr/wsgi.py

```diff
@@ -17,16 +17,14 @@
 from os.path import abspath, dirname
 from sys import path
 
-import dotenv
+from django.core.exceptions import ImproperlyConfigured
 from django.core.wsgi import get_wsgi_application
 
 SITE_ROOT = dirname(dirname(abspath(__file__)))
 path.append(SITE_ROOT)
 
-dotenv.read_dotenv(
-    os.path.join(os.path.dirname(os.path.dirname(__file__)), ".env")
-)
```

Comment:
I'm assuming we don't need to read `.env` in `wsgi.py` as we'll only run this inside the container. Inside the container the env vars are set by the orchestrator. For a local install I'm assuming we will use `manage.py runserver` so we'll read `.env` in `manage.py` if it is present.